### ISSUE TITLE
Error when using docker registry password longer than 1000 chars (for example Amazon Elasstic Container Registry tokens)

### ISSUE THREAD

**Issue by MatthiasWinzeler** [cloudfoundry/cloud_controller_ng#1141]
## Issue
Pushing a Docker application with a private Docker registry using a password of more than 1000 characters results in a server-side error.

## Context
https://stackoverflow.com/questions/49015047/cannot-deploy-docker-image-form-aws-private-registry

Cloud controller's log shows the following:
```
/var/vcap/sys/log/cloud_controller_ng/cloud_controller_ng.log.5.gz:{"timestamp":1526311559.8367982,"message":"Request failed: 500: {\"error_code\"=>\"UnknownError\", \"description\"=>\"An unknown error occurred.\", \"code\"=>10001, \"test_mode_info\"=>{\"description\"=>\"docker_password can be up to 1,000 characters\",
```

From inspecting the cloud controller code, it looks that the limit was reduced from 16000 chars to 1000 chars: https://github.com/cloudfoundry/cloud_controller_ng/commit/e06d71dfaa0bcb04dd3916f1199d33eb0fa74716

The user in this post mentioned to us that
> We are using the Amazon Elastic Container Registry and the passwords generated are really long (around 2k characters) because they are signed tokens.
So I guess there are valid use cases for these long passwords.

Is there a reason for this limit? Could we increase it?

**Comment by zrob** [comment]
Actually the limit was around 255 before, I know that commit is confusing, but it came from this issue https://github.com/cloudfoundry/cloud_controller_ng/issues/843.

It seems reasonable to bump it again.

**Comment by zrob** [comment]
should be resolved in linked story and available in the next capi-release 1.61.0

please re-open if the issue persists after that version
